src/components/ContentModal.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="modal">
+    <div class="modal-content" @click.stop>
+      <p id="modalText" v-if="!isImage">
+        {{ cardContent.content }}
+      </p>
+      <img
+        alt="image"
+        id="modalImage"
+        :src="cardContent.content"
+        class="img-responsive"
+        v-if="isImage"
+      />
+      <slot></slot>
+    </div>
+    <div class="modal-backdrop"></div>
+  </div>
+</template>
+
+<script lang="ts">
```

Comment:
use script setup

Reply:
I changed it to composition api and script setup in my latest commit